src/app/dashboard/dashboard.component.css

```diff
@@ -0,0 +1,54 @@
+/* DashboardComponent's private CSS styles */
+
+h2 {
+  text-align: center;
+}
+
+.heroes-menu {
+  padding: 0;
+  margin: auto;
+  max-width: 1000px;
+
+  /* flexbox */
+  display: -webkit-box;
```

Comment:
Shouldn't need to prefix these, the autoprefixer will sort it out. No need to comment as flexbox either.